packages/file_selector/file_selector_android/lib/file_selector_android.dart

```diff
@@ -78,9 +78,9 @@ class FileSelectorAndroid extends FileSelectorPlatform {
     return pathList?.map((String path) => XFile(path)).toList() ?? <XFile>[];
   }
 
-  /// We can't currently set the Confirm Button Text
-  /// For references, please check the following link
-  /// https://developer.android.com/reference/android/content/Intent#ACTION_OPEN_DOCUMENT_TREE
+  // We can't currently set the Confirm Button Text
```

Comment:
Why don't we use `///`?
Try to avoid `we` -_-

Reply:
`///` are used as an api comment, `//` is for informative purposes to developers of this package

What do you suggest for `we`? `Android does not currently support...` etc?

Reply:
Yep